twitch4j/src/main/java/com/github/twitch4j/TwitchClientBuilder.java

```diff
@@ -288,12 +291,34 @@ public class TwitchClientBuilder {
 
     /**
      * Websocket Close Delay in ms (0 = minimum)
-
+     *
      * @see WebsocketConnectionConfig#closeDelay()
      */
     @With
     private int wsCloseDelay = 1_000;
 
+    /**
+     * Whether chat commands should be executed via the Helix API, if possible.
+     * <p>
+     * Must have {@link #withEnableHelix(Boolean)} and {@link #withEnableChat(Boolean)} set to true.
+     * <p>
+     * Must have {@link #withChatAccount(OAuth2Credential)} or {@link #withDefaultAuthToken(OAuth2Credential)} specified.
+     */
+    @With
+    private boolean chatCommandsViaHelix = Instant.now().isAfter(Instant.ofEpochSecond(1676678400L));
```

Comment:
This corresponds to `Saturday, February 18, 2023 12:00:00 AM GMT`

While this can be adjusted later (e.g., if details about rolling sunset windows are released), do we want to toggle the forwarder early since this condition is only evaluated on initial client construction... if so, early by 1 day? 1 week?

Reply:
I think one week early would be a good.